model/accounts_mgmt/v1/feature_toggle_query_resource.model

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages feature toggle query.
+resource FeatureToggleQuery {
+	// Retrieves the details of the feature toggle by providing query context
```

Comment:
This is a `POST` request, but "Retrieves"? The generated SDK will lists it as responding with "201 - Created"

Reply:
Oh great, I was wondering about it. How can I change it to be 200?

Reply:
It's automatically generated through the metamodel. I don't think we have a way of coercing the response code to be different.

Only `GET` requests respond with 200...